hw.py

```diff
@@ -0,0 +1,65 @@
+import requests
+
+
+class Pet:
+
+    BaseUrl = 'https://petstore.swagger.io/v2/pet'
+
+    @staticmethod
+    def checker(response):
+        """Check response on valid"""
+        try:
+            response.raise_for_status()
+            try:
+                response.json()
```

Comment:
```suggestion
                return response.json()
```
ну так логічніше, як помилки не буде то поверне джсон

Reply:
Так, спочатку в мене і була така реалізація, та потім я подумав, що ніколи в try/except не юзаю елсе, то хай буде) Обидва варіанти правильні все одно)